recipes/zppy/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "zppy" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/zppy/archive/v{{ version }}.tar.gz
+  sha256: a233512d612bef036f022b40c9b469eb6d28f6cfcf8f16ebf6dcae946df49d0a
+
+build:
+  entry_points:
+    - zppy=zppy.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

Following https://github.com/E3SM-Project/zppy/blob/main/conda/meta.yaml, this should be:
```suggestion
    - python >=3.6
    - configobj >=5.0.0,<6.0.0
    - jinja2 >=2.0.0
    - mache >=1.3.2
    - mpas_tools >=0.15.0
```
I think some of these are missing from `setup.py` largely because they're not on PyPI.

---

@forsyth2 and @tomvothecoder, same question, is this not where you keep the requirements you use to build the conda package now?